app/helpers.php

```diff
@@ -93,3 +95,40 @@ function getRtlCss($path)
         return implode('/', $path).'/'.$filename.'.rtl.css';
     }
 }
+
+if (! function_exists('redirectToHome')) {
+    /**
+     * Redirect to the "home" page depending on authentication/authorization status.
+     *
+     * @param $flashMsg
+     * @param $$arrData
+     */
+    function redirectToHome($flashMsg = null, $arrData = null)
+    {
+        if (access()->allow('view-backend')) {
+            return redirect()->route('admin.dashboard')->withFlashDanger($flashMsg, $arrData);
+        } elseif (Auth::check()) {
+            return redirect()->route('frontend.user.dashboard')->withFlashDanger($flashMsg, $arrData);
+        }
+
+        return redirect()->route('frontend.index')->withFlashDanger($flashMsg, $arrData);
+    }
```

Comment:
I'm still not 100% sure I'd need this feature myself, but I'm not against the idea. 
From a code review side though, you're duplicating the `homeRoute()` Logic. I'm sure you could change redirectToHome to this and it would do the same thing. 

```
function redirectToHome($flashMsg = null, $arrData = null)
    {
        return redirect()->homeRoute()->withFlashDanger($flashMsg, $arrData);
    }
  ```